apps/rabbitmq.go

```diff
@@ -29,17 +29,28 @@ func (*LoggingProcessorRabbitmq) Type() string {
 }
 
 func (p *LoggingProcessorRabbitmq) Components(tag, uid string) []fluentbit.Component {
-	c := []fluentbit.Component{}
-	regexParser := confgenerator.LoggingProcessorParseRegex{
-		// Sample log line:
-		// 2022-01-31 18:01:20.441571+00:00 [erro] <0.692.0> ** Connection attempt from node 'rabbit_ctl_17@keith-testing-rabbitmq' rejected. Invalid challenge reply. **
-		Regex: `^(?<timestamp>\d+-\d+-\d+ \d+:\d+:\d+\.\d+\+\d+:\d+) \[(?<severity>\w+)\] \<(?<process_id>\d+\.\d+\.\d+)\> (?<message>.*)$`,
-		ParserShared: confgenerator.ParserShared{
-			TimeKey:    "timestamp",
-			TimeFormat: "%Y-%m-%d %H:%M:%S.%L%z",
+	c := confgenerator.LoggingProcessorParseRegexComplex{
+		Parsers: []confgenerator.RegexParser{
+			{
+				// Sample log line:
+				// 2022-01-31 18:01:20.441571+00:00 [erro] <0.692.0> ** Connection attempt from node 'rabbit_ctl_17@keith-testing-rabbitmq' rejected. Invalid challenge reply. **
+				Regex: `^(?<timestamp>\d+-\d+-\d+ \d+:\d+:\d+\.\d+\+\d+:\d+) \[(?<severity>\w+)\] \<(?<process_id>\d+\.\d+\.\d+)\> (?<message>.*)$`,
```

Comment:
```suggestion
				Regex: `^(?<timestamp>\d+-\d+-\d+\s+\d+:\d+:\d+[.,]\d+\+\d+:\d+) \[(?<severity>\w+)\] \<(?<process_id>\d+\.\d+\.\d+)\> (?<message>.*)$`,
```

(might not be needed, haven't actually checked; this is just to guard against the possibility of 1) hours having a leading space instead of leading zero for padding when less than 10, and 2) decimal separator not being `.` in all locales)

Reply:
Updated [regex](https://regex101.com/r/Tb0Ya3/1)

Reply:
Can you update the first regex too?